### PR TITLE
about-gcc: adding gcc description in the about-organization page

### LINKED ISSUE #232: gcc: add gcc links on prologin site

### LINKED ISSUE #250: Update about information
The https://prologin.org/about/organization page should be updated to add information about Girls Can Code (and a link redirecting there perhaps?).

### PR DESCRIPTION
**Pull request by Elfikurr**
Closes #250 

Hello,

This PR simply adds a Girls Can Code! description with a link to the Girls Can Code! website on the `about-organization` page.

Feel free to update, correct or improve the description if needed.

Thanks in advance for your reviews.

### CODE REVIEW

File: prologin/pages/templates/pages/about-organization.html [hunk 1]
```diff
@@ -94,6 +94,26 @@ <h2 id="organization">{% trans "A computer enthusiasts organization" %}</h2>
 with us!</p>
 {% endblocktrans %}
 
+{% url 'pages:about-organization' as about_organization %}
+
+<h2 id="gcc">{% trans "Girls Can Code!" %}</h2>
+{% blocktrans %}<p class="text-justify">Prologin started to organize in 2014 the Girls Can Code! summer camps.
+These training camps are the perfect occasion for young girls who are still in middle school or in high school 
+to discover multiple computer science subjects throughout the week. They will learn how to solve problems with 
+algorithms, explore numerous material such as network, web development, robotics, microcontrollers, video games
```
Comment (zopieux) [inline prologin/pages/templates/pages/about-organization.html]: s/material/CS topics/ or something

File: prologin/pages/templates/pages/about-organization.html [hunk 2]
```diff
@@ -94,6 +94,26 @@ <h2 id="organization">{% trans "A computer enthusiasts organization" %}</h2>
 with us!</p>
 {% endblocktrans %}
 
+{% url 'pages:about-organization' as about_organization %}
+
+<h2 id="gcc">{% trans "Girls Can Code!" %}</h2>
+{% blocktrans %}<p class="text-justify">Prologin started to organize in 2014 the Girls Can Code! summer camps.
```
Comment (zopieux) [inline prologin/pages/templates/pages/about-organization.html]: In 2014, Prologin …

File: prologin/pages/templates/pages/about-organization.html [hunk 3]
```diff
@@ -94,6 +94,26 @@ <h2 id="organization">{% trans "A computer enthusiasts organization" %}</h2>
...
+These training camps are the perfect occasion for young girls who are still in middle school or in high school 
+to discover multiple computer science subjects throughout the week. They will learn how to solve problems with 
+algorithms, explore numerous material such as network, web development, robotics, microcontrollers, video games 
+and more through practical exercises.</p>
+
+<p class="text-justify">Conferences from exterior speakers are organized for every camp. It enables the attendees 
+to exchange with women working in the computer science industry or in research fields. The young girls are sometimes,
```
Comment (zopieux) [inline prologin/pages/templates/pages/about-organization.html]: s/or in research fields/or acadamia/ maybe?

Comment (zopieux) [inline prologin/pages/templates/pages/about-organization.html]: extraneous trailing comma

File: prologin/pages/templates/pages/about-organization.html [hunk 4]
```diff
@@ -94,6 +94,26 @@ <h2 id="organization">{% trans "A computer enthusiasts organization" %}</h2>
...
+<h2 id="gcc">{% trans "Girls Can Code!" %}</h2>
+{% blocktrans %}<p class="text-justify">Prologin started to organize in 2014 the Girls Can Code! summer camps.
+These training camps are the perfect occasion for young girls who are still in middle school or in high school 
+to discover multiple computer science subjects throughout the week. They will learn how to solve problems with 
+algorithms, explore numerous material such as network, web development, robotics, microcontrollers, video games 
+and more through practical exercises.</p>
+
+<p class="text-justify">Conferences from exterior speakers are organized for every camp. It enables the attendees 
+to exchange with women working in the computer science industry or in research fields. The young girls are sometimes, 
+unsure as to what they want to do, this is a golden occasion for them to discuss with the speaker their educational
```
Comment (zopieux) [inline prologin/pages/templates/pages/about-organization.html]: missing "about" or some glue word after "discuss with"

File: prologin/pages/templates/pages/about-organization.html [hunk 5]
```diff
@@ -94,6 +94,26 @@ <h2 id="organization">{% trans "A computer enthusiasts organization" %}</h2>
...
+to discover multiple computer science subjects throughout the week. They will learn how to solve problems with 
+algorithms, explore numerous material such as network, web development, robotics, microcontrollers, video games 
+and more through practical exercises.</p>
+
+<p class="text-justify">Conferences from exterior speakers are organized for every camp. It enables the attendees 
+to exchange with women working in the computer science industry or in research fields. The young girls are sometimes, 
+unsure as to what they want to do, this is a golden occasion for them to discuss with the speaker their educational 
+backgrounds, thus helping them for their orientation.</p>
+
+<p class="text-justify">Prologin's members are present during the camp to accompany them and to help them debug their
```
Comment (zopieux) [inline prologin/pages/templates/pages/about-organization.html]: *Prologin staff is present*

Comment (zopieux) [inline prologin/pages/templates/pages/about-organization.html]: Too many *and*s. Use the form "x, y and z".

File: prologin/pages/templates/pages/about-organization.html [hunk 6]
```diff
@@ -94,6 +94,26 @@ <h2 id="organization">{% trans "A computer enthusiasts organization" %}</h2>
...
+
+<p class="text-justify">Conferences from exterior speakers are organized for every camp. It enables the attendees 
+to exchange with women working in the computer science industry or in research fields. The young girls are sometimes, 
+unsure as to what they want to do, this is a golden occasion for them to discuss with the speaker their educational 
+backgrounds, thus helping them for their orientation.</p>
+
+<p class="text-justify">Prologin's members are present during the camp to accompany them and to help them debug their 
+programs and to help them understand key principles in computer science and programming.</p>
+
+<p class⁼"text-justify">More information can be found on Girls Can Code! <a href="https://gcc.prologin.org">website</a>. For any
```
Comment (zopieux) [inline prologin/pages/templates/pages/about-organization.html]: Include GCC in the link

File: prologin/pages/templates/pages/about-organization.html [hunk 7]
```diff
@@ -94,6 +94,26 @@ <h2 id="organization">{% trans "A computer enthusiasts organization" %}</h2>
...
+<p class="text-justify">Conferences from exterior speakers are organized for every camp. It enables the attendees 
+to exchange with women working in the computer science industry or in research fields. The young girls are sometimes, 
+unsure as to what they want to do, this is a golden occasion for them to discuss with the speaker their educational 
+backgrounds, thus helping them for their orientation.</p>
+
+<p class="text-justify">Prologin's members are present during the camp to accompany them and to help them debug their 
+programs and to help them understand key principles in computer science and programming.</p>
+
+<p class⁼"text-justify">More information can be found on Girls Can Code! <a href="https://gcc.prologin.org">website</a>. For any 
+inquiries, please contact us <a href="{{ about_organization }}#contact">here</a>.</p>{% endblocktrans %}
```
Comment (zopieux) [inline prologin/pages/templates/pages/about-organization.html]: please [contact us].

**Review by zopieux — CHANGES_REQUESTED** [review]

**Review by fnareoh — CHANGES_REQUESTED** [review]
I approve the English content but I would like to check the translation before approving for merging. Thank you for your work, it really needed to be added !

**Review by fnareoh — APPROVED** [review]

**Comment by Elfikurr** [comment]
I'll merge this soon, and I'll update the translation on Transifex when doing so.

**Pull request merged**